extras/jenkins/AWS/Jenkinsfile

```diff
@@ -156,8 +156,6 @@ pipeline {
             $WORKSPACE/pulumi/python/venv/bin/pulumi config set prometheus:helm_timeout "600" -C pulumi/python/config -s marajenkaws${BUILD_NUMBER}
             $WORKSPACE/pulumi/python/venv/bin/pulumi config set aws:profile "${AWS_PROFILE}" -C pulumi/python/config -s marajenkaws${BUILD_NUMBER}
             $WORKSPACE/pulumi/python/venv/bin/pulumi config set aws:region "${AWS_DEFAULT_REGION}" -C pulumi/python/config -s marajenkaws${BUILD_NUMBER}
-            $WORKSPACE/pulumi/python/venv/bin/pulumi config set sirius:accounts_pwd --secret "Password" -C pulumi/python/kubernetes/applications/sirius -s marajenkaws${BUILD_NUMBER}
-            $WORKSPACE/pulumi/python/venv/bin/pulumi config set sirius:ledger_pwd --secret "Password" -C pulumi/python/kubernetes/applications/sirius -s marajenkaws${BUILD_NUMBER}
```

Comment:
Question: Why are these being removed?  It wasn't clear to me based on my read of the PR description.

Reply:
So, this is why Jenkins didn't catch it - because I don't want to play around with expect or another way of driving the startup script I pre-populate a bunch of the config data. Including....you guessed it, those two passwords. Which you can still set yourself in this file if you want. This means that bit of code above that we missed never got executed until I went and manually ran things.

This does point towards the need for whatever replaces these scripts to have its own test harness.